Tasks/AzurePowerShellV2/TryMakingModuleAvailable.ps1

```diff
@@ -0,0 +1,33 @@
+[CmdletBinding()]
```

Comment:
Instead of repeating this for each task, you can make this logic more generalise for Az modules as well and move to common folder

Reply:
@AmrutaKawade can you please point to the right place where we should move this.

Reply:
this can be moved here https://github.com/microsoft/azure-pipelines-tasks/tree/master/Tasks/Common/VstsAzureHelpers_

Reply:
I had checked, though we would want to do it but the other code that the new one depends on resides in Utility and that is not available in Common. So to implement that we would need to refactor the existing code, so I'll have to skip this comment.